actions/action-lint/README.md

```diff
@@ -0,0 +1,46 @@
+# helm-lint
```

Comment:
Is it really Helm related? To me looks like GitHub Actions lint

Reply:
oh yes you're right .i just updated it

Reply:
this still has the wrong title